ansible/setup.yml

```diff
@@ -98,9 +105,15 @@
         - ./configure
         - make
         - make install
-        - ldconfig
       args:
         chdir: "/usr/local/src/libsodium-{{ libsodium_version }}"
         creates: "/usr/local/lib/libsodium.so.{{ libsodium_module_version }}"
       tags:
         - libsodium
+
+    - name: Run ldconfig
+      command: ldconfig
+      args:
+        creates: /etc/ld.so.cache
```

Comment:
@dincho ok so can you give me a tip how to  make this command, that it will not fail with ansible-lint?

```
[ANSIBLE0012] Commands should not change things if nothing needs doing
ansible/setup.yml:114
```

Reply:
Good point, might be using `when: installed` should solve it, where `installed` should be registered in the previous task.